### ISSUE TITLE
Identify Breadcrumbs - iOS

### ISSUE THREAD

**Issue by ZoeSimon** [bcgov/lucy-web#507]
**Describe the Task**
Identify Breadcrumbs for iOS.

Breadcrumbs in this context include: "This exists, it is blocked, here is why, and this is what to do when it is unblocked"

**Comment by ghost** [comment]
@AmirShayegh or @fwpushan update the doc below to include ios breadcrumbs

https://docs.google.com/document/d/1rF9MTJzGIFVv1Ds0yX4TiwAvE9PTJvPscbyxhCWW7-s/edit#

**Comment by ZoeSimon** [comment]
Mike Wells' comment from ticket #495 

- what things are partly done
- what low priority stuff has no tests, but needs doc
- things that were not done, but anticipated to be needed later, and advice on changes that would needed to be made, such as:
- what things would need to change if we moved to high availability i.e. more than one pod running your api.

**Comment by ZoeSimon** [comment]
@micheal-w-wells Review the document in this ticket please :)

**Comment by micheal-w-wells** [comment]
It is useful certainly but what it fails to highlight is what is partially done vs not done, it is a big list of things but doesn't indicate what progress if any has been made on them.

**Comment by micheal-w-wells** [comment]
I'll clarify that there are some scattered notes on progress on a few items, but some have no info on what if anything has been done.
